src/hooks.ts

```diff
@@ -14,42 +14,59 @@ export function useResource<T>(resource: Resource<T>): UseResourceResult<T> {
     loading: true,
     error: null,
     // These are lies to make the API nicer to use.
-    body: null as any,
-    state: null as any,
+    data: null as any,
+    resourceState: null as any,
```

Comment:
What is the difference between `data` and `resourceState`? is `resourceState` the full resource?

Reply:
`.get()` returns a `State`, and data is the same as `State.data`.

So if you only care about, say, the properties (like we have in videodb), you can just use `data`. If you want the full state object you can use `state`.

So I think for hooks I would expect someone to do:

`const loading, error, data} = useResource()`

for read-only cases, and 

`const {loading, error, resourceState} = useResource()`

for more complex cases.

Or use both. The options are there.

Reason I went for `resourceState` vs just `state` is because I was worried that calling things `state` is going to get really confusing for react developers.